### PR TITLE
UnixPB: Update Debian URLs

### LINKED ISSUE #3173: Ansible - Debian Repository URLs are incorrect.
URLs for Debian GPG key and AdoptopenJDK repository require updating to packages.adoptium.net 

Delete as appropriate from this list:

- Bug in ansible playbook

Details:

The existing URLs for the GPG key and Debian repository are incorrect and require updating.

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3173 

Update URLs for GPG and repository to adoptium.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

### CODE REVIEW

**Comment by steelhead31** [comment]
VPC check in progress: https://ci.adoptium.net/job/VagrantPlaybookCheck/1706/

**Review by karianna — APPROVED** [review]

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml [hunk 1]
```diff
@@ -25,13 +25,13 @@
 
 - name: Add AdoptOpenJDK GPG key
```
Comment (gdams) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml:26]:
```suggestion
- name: Add Eclipse Temurin GPG key
```

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
   apt_key:
-    url: https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+    url: https://packages.adoptium.net/artifactory/api/gpg/key/public
     state: present
   tags: patch_update
 
 - name: Add the adoptopenjdk repository to apt for JDK8
```
Comment (gdams) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml:32]:
```suggestion
- name: Add the Eclipse Temurin repository to apt for JDK8
```

**Review by gdams — COMMENTED** [review]
Two nits

**Review by adamfarley — APPROVED** [review]
LGTM - Does this mean the file folder will be renamed to Adoptium too?

**Comment by steelhead31** [comment]
> LGTM - Does this mean the file folder will be renamed to Adoptium too?

Not at this stage, this PR simply points the URL at their correct location, as the old ones no longer work, everything else should remain the same ( for now! )

**Review by gdams — CHANGES_REQUESTED** [review]
Just realised, you'll need to change https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Debian.yml#L71 too

**Comment by steelhead31** [comment]
> Just realised, you'll need to change https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Debian.yml#L71 too
> 
> Switch it to `temurin-8-jdk`

Good spot, done!

**Review by gdams — APPROVED** [review]

**Pull request merged**